Source/WebKit/WebProcess/WebCoreSupport/WebPermissionController.cpp

```diff
@@ -88,7 +88,7 @@ void WebPermissionController::updateCache(const WebCore::ClientOrigin& origin, c
 
 void WebPermissionController::tryProcessingRequests()
 {
-    if (m_requests.isEmpty() || !m_page)
+    if (m_requests.isEmpty())
```

Comment:
You can remove this, the while loop won't run if `m_requests` is empty

Reply:
Fixed!